lib/data_cycle/update_logger.rb

```diff
@@ -4,9 +4,9 @@ class UpdateLogger
   ################
   # Entry points #
   ################
-  def self.update_course(course, times)
+  def self.update_course(course, hash_info)
```

Comment:
Can you think of a better name than `hash_info`? I tried to leave a comment before but it might not have gone through...

I think you could call this `new_logs`, and then rename the intermediate variable below to `updated_logs`. That makes it a little less generic and reduces the chance for someone getting confused about what this is supposed to do.

Reply:
Done!